
uint8 left_speed
uint8 left_dir
uint8 right_speed
uint8 right_dir
